src/views/Profile.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-title>{{ $t("Profile") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
```

Comment:
@azkyakhan please make sure to check all PRs for full screen true

Reply:
Removed it sir.